### PR TITLE
Increase memory on iOS

### PR DESCRIPTION
**Pull request by mozzius**
Adds increased memory limit entitlement

### CODE REVIEW

**Review by gaearon — APPROVED** [review]
let's try it

File: app.config.js [hunk 1]
```diff
@@ -99,6 +99,8 @@ module.exports = function (config) {
           dark: DARK_SPLASH_CONFIG,
         },
         entitlements: {
+          'com.apple.developer.kernel.increased-memory-limit': true,
+          'com.apple.developer.kernel.extended-virtual-addressing': true,
```
Comment (hirbod) [inline app.config.js:103]: I don't think you need this, but it doesn't hurt

**Comment by hirbod** [comment]
In case you wanna bump Android too, here is a config plugin (I highly recommend it):

```js
const { withAndroidManifest } = require('@expo/config-plugins')

module.exports = function withAndroidLargeHeap(config) {
  return withAndroidManifest(config, async (config) => {
    config.modResults.manifest.application[0].$['android:largeHeap'] = 'true'

    return config
  })
}
```

**Comment by mozzius** [comment]
@hirbod I found we already have it! https://github.com/bluesky-social/social-app/blob/main/plugins/withAndroidManifestPlugin.js

**Pull request merged**